### DISCUSSION TITLE
Digiblast (Nikko) portable console cartridge pinout.

### DISCUSSION THREAD

**Discussion by buggii** [sanni/cartreader#942] [Ideas & Suggestions]
32 pin GameBoy-like cartridge:

 1: VCC
 2: VCC
 3: VCC
 4: Pull up/down?
 5: NAND R/B#
 6: GND
 7: NAND IO8
 8: NAND IO7
 9: NAND IO6
10: NAND IO5
11: GND
12: NAND IO4
13: NAND IO3
14: NAND IO2
15: NAND IO1
16: GND
17: NAND ALE
18: NAND WE#
19: NAND RE#
20: NAND CE#
21: NAND CLE
22: GND
23: N/C
24: N/C
25: USB DM
26: USB DP
27: GND
28: EEPROM SCL
29: EEPROM SDA
30: GND
31: VBUS
32: GND

By Cyanic

**Comment by Ancyker** [comment]
Possible adapter for it:

[snes-to-db_rev1_gerber.zip](https://github.com/sanni/cartreader/files/15210175/snes-to-db_rev1_gerber.zip)

If someone confirms it works/writes the firmware core I will commit the KiCad project.

![image](https://github.com/sanni/cartreader/assets/7105333/4f66aec2-18ba-4c4a-9850-5f637b30a7dd)

**Reply by Ancyker** [reply]
Here are the data sheets for the 2 chips the OP has carts for:

[DS0043DMTSOPR1.11_062405.pdf](https://github.com/sanni/cartreader/files/15210202/DS0043DMTSOPR1.11_062405.pdf)

[K9F1G08Q0M-YCB0.PDF](https://github.com/sanni/cartreader/files/15210203/K9F1G08Q0M-YCB0.PDF)

**Reply by Ancyker** [reply]
If no one has a cart to test with and knows how to make a new core then if someone who has a cart gets an adapter made I will try to write the core blind.

**Reply by gorgyrip** [reply]
Thank you for the matrix datasheet. Years ago, I searched for it, but i couldn't find it. I ended up removing the soc and I followed the traces from the pads to the cart connector :)) I dumped so far 8 carts.
EDIT: you don't need cart pin 4 to dump a cart. And cart pin 3 is not vcc inside the console You can also use a gba connector, but you have to cut the plastic from the sides.

**Reply by Ancyker** [reply]
@gorgyrip I had the PCB produced, but I do not have a cartridge to test it with. If you have a cart I could borrow, then I could test it. Or I can send you an adapter for free, just pay shipping (likely around US$5 domestic [USA] or $15~20 international). Until then, without a cart, I can't test the adapter.

As I live in the USA, these carts aren't readily available here. Most of the listings are located in Italy, with shipping costing between US$30 and $200+...

![Screenshot of Digiblast listings on eBay with highlighted shipping costs ranging from $30 to $209](https://github.com/user-attachments/assets/d031e8ed-4c6d-45a2-8852-6cafbe3b1710)

There does appear to be one on eBay located in the US for about $40, which is quite a lot for a game that's for a system I will likely never play/own.

**Reply by gorgyrip** [reply]
I ended up here looking for digiblast carts, not because of the reader.  So your adapter is no use for me, I just wanted to share some info. I'm aiming as dumping as many as possible. I think i will have 5 new titles soon.
I'm from Romania. I gave up ebay selling, too much stress. But i sold 2 carts for about 11gbp , shipping about 7 gbp, anywhere in the world as a registered letter. and still have many winx carts, I'll let you know if/when I decide to sell again.
